partner_delivery_schedule/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Delivery Schedule',
+    'summary': 'Set on partners a schedule for delivery goods',
+    'version': '11.0.1.0.0',
+    'development_status': 'Production/Stable',
```

Comment:
It should be `Beta` when the module first arrives. It gets to `Production` when it's present in 2 versions. See https://odoo-community.org/page/module-maturity-levels